inference/utils/download_llama_weights.py

```diff
@@ -8,6 +8,9 @@
 # You can pass the --use-full-precision flag to use the full-precision weight. By default, we use half precision.
 parser = argparse.ArgumentParser()
 parser.add_argument("--use-full-precision", action="store_true", help="Use full precision")
+parser.add_argument("--use_13B", action="store_true", help="Use full precision")
```

Comment:
@DerrickYLJ  could we also update the help descriptions for these three new arguments here?